lldb/include/lldb/Utility/Stream.h

```diff
@@ -260,10 +271,9 @@ class Stream {
   ///     The ANSI color code to end colorization. This is
   ///     environment-dependent.
 
-  void PutCStringColorHighlighted(llvm::StringRef text,
-                                  llvm::StringRef pattern = "",
-                                  llvm::StringRef prefix = "",
-                                  llvm::StringRef suffix = "");
+  void PutCStringColorHighlighted(
+      llvm::StringRef text,
+      std::optional<HighlightSettings> pattern_info = std::nullopt);
```

Comment:
`pattern_info` relates to the old way of doing it more than this.

How about `settings`? It will be obvious that it's settings for highlighting because it's in `PutCStringColorHighlighted`.

Reply:
✅